playbooks/custom_certificates.yml

```diff
@@ -0,0 +1,4 @@
+---
+- hosts: all
+  roles:
```

Comment:
Should there be `become: true` here? 

fatal: [centos7-katello-nightly]: FAILED! => {"changed": false, "failed": true, "msg": "There was an issue creating /root/custom_certificates as requested: [Errno 13] Permission denied: '/root/custom_certificates'", "path": "/root/custom_certificates/certs", "state": "absent"}